lib/ansible/modules/cloud/ovirt/ovirt_datacenter.py

```diff
@@ -65,6 +65,12 @@
             - "IMPORTANT: This option is deprecated in oVirt/RHV 4.1. You should
                use C(mac_pool) in C(ovirt_clusters) module, as MAC pools are
                set per cluster since 4.1."
+    force:
+        description:
+            - "If True DC will be forcibly moved to absent state. Valid when state is I(absent)"
```

Comment:
The description is not clear, I'd rephrase:
"This parameter can be used only when removing a data center. If I(True) data center will be forcibly removed, even though it contains some clusters. Default value is I(False), which means that only empty data center can be removed."

Reply:
@mwperina Thanks, reposted.